File: api-tester/src/latestDependencies/java/com/revenuecat/apitester/java/PurchasesBC4API.java
package com.revenuecat.apitester.java;

import com.revenuecat.purchases.BillingFeature;

@SuppressWarnings({"unused"})
final class PurchasesBC4API {
    static void check(final BillingFeature feature) {
        switch (feature) {
            case SUBSCRIPTIONS:
            case SUBSCRIPTIONS_UPDATE:
            case PRICE_CHANGE_CONFIRMATION:
            case IN_APP_ITEMS_ON_VR:
            case SUBSCRIPTIONS_ON_VR:
        }
    }
}

File: api-tester/src/unityIAP/java/com/revenuecat/apitester/kotlin/PurchasesBC4API.kt
package com.revenuecat.apitester.kotlin

import com.revenuecat.purchases.BillingFeature

@Suppress("unused")
private class PurchasesBC4API {
    fun check(billingFeature: BillingFeature) {
        when (billingFeature) {
            BillingFeature.PRICE_CHANGE_CONFIRMATION,
            BillingFeature.SUBSCRIPTIONS,
            BillingFeature.SUBSCRIPTIONS_UPDATE,
            BillingFeature.IN_APP_ITEMS_ON_VR,
            BillingFeature.SUBSCRIPTIONS_ON_VR,
            -> {}
        }.exhaustive
    }
}
